full_course/notebooks/cf-python_regridding.ipynb

```diff
@@ -1193,9 +1197,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.6"
+   "version": "3.12.0"
```

Comment:
I will shorty check that both Notebooks work under the JASMIN Notebook service which uses the default JasPy environment, which [currently is `jaspy/3.11/v20240815`](https://github.com/cedadev/ceda-jaspy-envs/blob/main/environments/py3.11/mf3-23.11.0-0/jaspy3.11-mf3-23.11.0-0-v20240815/final-spec.yml) notably with Python 3.11 and the latest cf* family releases, so the 3.12 and 3.11 Python version should be the only major difference but I am not expecting any issues.